sdk/storage/storage-file-datalake/src/models.ts

```diff
@@ -619,9 +631,76 @@ export interface FileFlushOptions extends CommonOptions {
   pathHttpHeaders?: PathHttpHeaders;
 }
 
-export interface FileCreateOptions extends PathCreateOptions {}
+export interface FileCreateOptions extends PathCreateOptions { }
+
+export interface FileCreateResponse extends PathCreateResponse { }
 
-export interface FileCreateResponse extends PathCreateResponse {}
+export interface FileParallelUploadOptions extends CommonOptions {
+  // For all.
+  abortSignal?: AbortSignalLike;
+  conditions?: DataLakeRequestConditions;
+
+  // For create and flush.
+  pathHttpHeaders?: PathCreateHttpHeaders;
+
+  // For create.
+  metadata?: Metadata;
+  permissions?: string; // TODO: model or string?
```

Comment:
might worth mention does before review, i see this TODO in existing functions, and it's a good time to finalize this before GA

Reply:
Copied from `PathCreateOptions`.
@XiaoningLiu what's the history and what do you suggest?
```
export interface PathCreateOptions extends CommonOptions {
  abortSignal?: AbortSignalLike;
  metadata?: Metadata;
  permissions?: string; // TODO: model or string?
  umask?: string; // TODO: model or string?
  conditions?: DataLakeRequestConditions;
  pathHttpHeaders?: PathCreateHttpHeaders;
}
```

Reply:
Will use `string` for now as we discussed.